schema.go

```diff
@@ -8,30 +8,30 @@ import (
 // Schema reflects a schema from a Go value.
 //
 // A schema is itself HCL.
-func Schema(v interface{}) (*AST, error) {
-	ast, err := marshalToAST(v, true)
+func Schema(v interface{}, opt *MarshalOptions) (*AST, error) {
+	ast, err := marshalToAST(v, true, opt)
 	if err != nil {
 		return nil, err
 	}
 	return ast, nil
 }
 
 // MustSchema constructs a schema from a Go type, or panics.
-func MustSchema(v interface{}) *AST {
-	ast, err := Schema(v)
+func MustSchema(v interface{}, options ...MarshalOption) *AST {
+	ast, err := Schema(v, NewMarshalOptions(options...))
 	if err != nil {
 		panic(err)
 	}
 	return ast
 }
 
 // BlockSchema reflects a block schema for a Go struct.
-func BlockSchema(name string, v interface{}) (*AST, error) {
+func BlockSchema(name string, v interface{}, opt *MarshalOptions) (*AST, error) {
```

Comment:
This should be `options...MarshalOption`.

Reply:
totally makes sense. Done